docs: added 'Business Development Manager' career (#138)
api7/website: _posts/careers/en-US/business-development-manager.md

---

> title: Business Development Manager

Reviewer: Not sure if it's correct here

date: 2021-06-22 21:52:00
---

> ## Content

Reviewer 2: `Content` heading should be replaced with `Responsibilities` , to be more precise,

content of this looks good to me for overseas/foreign people too

> - According to the company's business development needs, find and explore cooperation resources that are beneficial to the company, and evaluate the cooperability of resources.

Reviewer: Hi @iamayushdas, is this file's contents clear to you?

> - Responsible for the management and maintenance of business customers, independently and efficiently complete business negotiations, follow-up, contract signing, cooperation landing, contract payment collection and payment, customer relationship maintenance and other work to achieve performance targets.

Reviewer 3: ```suggestion
- Responsible for the management and maintenance of business customers, independently and efficiently complete business negotiations, follow-up, contract signing, cooperation landing, contract payment collection and payment, customer relationship maintenance and other works to achieve performance targets.
```

> - Responsible for the maintenance of business customers, regular collection of audience feedback on the commercial product side and timely feedback to the company for product adjustment and optimization.

Reviewer 3: ```suggestion
- Responsible for the maintenance of business customers, regular collection of audience feedbacks on the commercial product side and timely feedback to the company for product adjustment and optimization.
```

> - Pay attention to the dynamic sales situation of business customers, sales data collection statistics and analysis. Collecting customer needs, providing professional support and services to customers, and making rationalized suggestions for their sales and operational situations.

Reviewer 3: ```suggestion
- Follow the dynamic sales situation of business customers, collect and analyze sales statistics. Collecting customer needs, providing professional support and services to customers, and making rationalized suggestions for their sales and operational situations.
```

- Complete other tasks as assigned by superiors.

## Requirements

- Bachelor's degree or above, good remote collaboration skills and the ability to use channel resources effectively.
- Understanding of open source culture and familiarity with products such as API gateways and load balancing products is a plus.
- Excellent business negotiation, coordination and marketing skills, able to complete marketing work independently. Ability to resist pressure, a sense of teamwork and strong execution.
- Flexible, meticulous, logical thinker with the ability to deal with emergencies.

## Additional

We're a remote work company with employees in 6 cities across China, and we do all of our collaboration through GitHub, Slack and Google Docs.

## Contact

> [wenming@api7.ai](mailto:wenming@api7.ai)

Reviewer: We will use this one

```suggestion
[support@api7.ai](mailto:support@api7.ai)
```

Author: OK, I see that the other files are still at that address and I will use a new PR to optimize the other files.
